src/components/Header/HeaderUserMenu/HeaderUserMenu.js

```diff
@@ -49,7 +49,7 @@ class HeaderUserMenu extends React.Component {
                 <strong>{user.plan.remainingCredits}</strong> {user.plan.description}
               </p>
             </div>
-            {user.hasClientManager && user.plan.buttonUrl && !user.plan.pendingFreeUpgrade ? (
+            {!user.hasClientManager && user.plan.buttonUrl && !user.plan.pendingFreeUpgrade ? (
```

Comment:
Did you verify other scenarios?

Maybe, before merging it, is a good idea waiting for some "Eagle Eyes" @CarlosSampedro's tests.

@fcoronelmakingsense could you add the links to testing this PR in different environments?

Reply:
im already put in comment, thanks!!